packages/babel-cli/src/babel/index.js

```diff
@@ -168,6 +169,9 @@ commander.option(
   "--delete-dir-on-start",
   "Delete the out directory before compilation",
 );
+
+commander.option("-S, --settings", "Show configuration settings");
```

Comment:
I'm not sure we should add a shorthand for this. S could be something else in the future. 

Note that this could also replace Babel --version?

Reply:
I'd prefer to keep `--version`, it still might be useful.

Reply:
Sorry I meant to say merging them not replacing.

Reply:
That was my point :)

Reply:
I had to misunderstand your intent then 😅